### PR TITLE
Side nav on sis filter refresh

### PR DESCRIPTION
**Pull request by yerramshilpa**
## Type of Change (Select One and Apply Github Label)

- [x] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [x] Internet Explorer 11
- [x] Edge
- [x] Chrome
- [x] Firefox
- [x] Safari

## Checklist:

- [x] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [x] My code passes the automated linter.
- [x] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { Component, ViewChild, AfterViewInit, ChangeDetectorRef, OnInit } from '@angular/core';
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts:1]: Remove unused dependencies

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <sds-accordion>
+    <sds-accordion-item  [expanded]="false">
+      <sds-accordion-item-header>
+           <span class="text-normal">Select Domain</span> 
+      </sds-accordion-item-header>
+      <sds-side-navigation #sideNav [model]="navigationModel"></sds-side-navigation>
+    </sds-accordion-item>
+    <sds-accordion-item #filtersAccordion>
+      <sds-accordion-item-header>Filter By</sds-accordion-item-header>
+      <sds-filters [form]="form" [fields]="fields" [model]="filterModel" (filterChange)="filterChange$.next($event)">
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.html:13]: unnecessary filterChange

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+              { label: '60 Days', value: '60' },
+              { label: '90 Days', value: '90' },
+
+            ]
+          },
+        },
+      ]
+    }
+  ];
+  ngOnInit() {
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts:63]: remove function -- results are not used anywhere

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+@Component({
+  templateUrl: 'sidenavigation-optional.component.html'
+})
+
+export class SideNavigationOptional implements OnInit  {
+  public navigationModel: SideNavigationModel = navigationConfig;
+  form:FormGroup;
+  filterModel: any;
+  results: any;
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts:17]: remove unused variable

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.module.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import { FormlyModule } from '@ngx-formly/core';
+
+@NgModule({
+  imports: [
+    SdsSideNavigationModule,
+    SdsAccordionModule,
+    CommonModule,
+    FormsModule,
+    SdsToolbarModule,
+    SdsPageModule,
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.module.ts:23]: remove unused import

Comment (yerramshilpa) [reply libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.module.ts]: Removed

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts (hunk 4 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts]: removed

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts (hunk 3 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts]: removed

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.html (hunk 2 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.html]: removed

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts (hunk 1 above)
Comment (yerramshilpa) [reply libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts]: removed

File: libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import { Component } from '@angular/core';
+import { SideNavigationModel} from '@gsa-sam/components'
+import { FormlyFieldConfig, FormlyFormOptions } from '@ngx-formly/core';
```
Comment (ascreven-gsa) [inline libs/documentation/src/lib/components/sidenavigation/demos/optional/sidenavigation-optional.component.ts:3]: FormlyFormOptions is not used

**Pull request merged**